project_event/security/ir.model.access.csv

```diff
@@ -1,8 +1,27 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_task_category,access_task_category,model_task_category,base.group_user,1,0,0,0
-access_task_category_manager,access_task_category_manager,model_task_category,project.group_project_manager,1,1,1,1
-access_event_template,access_event_template_group,model_event_template,base.group_user,1,1,1,1
-access_activity_template,access_activity_template_group,model_activity_template,base.group_user,1,1,1,1
-access_task_template,access_task_template_group,model_task_template,base.group_user,1,1,1,1
-access_res_partner_category_type,access_res_partner_category_type_group,model_res_partner_category_type,base.group_user,1,1,1,1
-access_res_partner_sector,access_res_partner_sector,model_res_partner_sector,base.group_user,1,1,1,1
+access_group_project_event_task_category_manager,access_group_project_event_task_category_manager,model_task_category,project_event.group_project_event_manager,1,1,1,1
+access_group_project_event_task_category_superuser,access_group_project_event_task_category_superuser,model_task_category,project_event.group_project_event_superuser,1,0,0,0
```

Comment:
superuser 1,0,0,0? shouldn't be 1,1,1,1

Reply:
@eilst according to the specification, the superuser has more access then the basic user and less access then the manager. In the spec superuser is equivalent to the editor group (check the editeur.ods file)